doc/guides/bluetooth/index.rst

```diff
@@ -3,310 +3,18 @@
 Bluetooth
 #########
 
-.. contents::
-    :local:
-    :depth: 2
+The Bluetooth section contains information regarding the Bluetooth stack of the
+Zephyr kernel. Use the information to understand the principles behind the
+operation of the layers and how they were implemented.
 
-Overview
-********
-
-Zephyr comes integrated with a feature-rich and highly configurable
-Bluetooth stack:
-
-* Bluetooth 5.0 compliant (ESR10)
-
-* Bluetooth Low Energy Controller support (LE Link Layer)
-
-  * BLE 5.0 compliant
-  * Unlimited role and connection count, all roles supported
-  * Concurrent multi-protocol support ready
-  * Intelligent scheduling of roles to minimize overlap
-  * Portable design to any open BLE radio, currently supports Nordic
-    Semiconductor nRF51 and nRF52
-
-* Generic Access Profile (GAP) with all possible LE roles
-
-  * Peripheral & Central
-  * Observer & Broadcaster
-
-* GATT (Generic Attribute Profile)
-
-  * Server (to be a sensor)
-  * Client (to connect to sensors)
-
-* Pairing support, including the Secure Connections feature from Bluetooth 4.2
-
-* Bluetooth Mesh support
-
-  * Relay, Friend Node, Low-Power Node (LPN) and GATT Proxy features
-  * Both Provisioning bearers supported (PB-ADV & PB-GATT)
-  * Highly configurable, fits as small as 16k RAM devices
-
-* IPSP/6LoWPAN for IPv6 connectivity over Bluetooth LE
-
-  * IPSP node sample application in ``samples/bluetooth/ipsp``
-
-* Basic Bluetooth BR/EDR (Classic) support
-
-  * Generic Access Profile (GAP)
-  * Logical Link Control and Adaptation Protocol (L2CAP)
-  * Serial Port emulation (RFCOMM protocol)
-  * Service Discovery Protocol (SDP)
-
-* Clean HCI driver abstraction
-
-  * 3-Wire (H:5) & 5-Wire (H:4) UART
-  * SPI
-  * Local controller support as a virtual HCI driver
-
-* Raw HCI interface to run Zephyr as a Controller instead of a full Host stack
-
-  * Possible to export HCI over a physical transport
-  * ``samples/bluetooth/hci_uart`` sample for HCI over UART
-  * ``samples/bluetooth/hci_usb`` sample for HCI over USB
-
-* Verified with multiple popular controllers
-
-* Highly configurable
-
-  * Features, buffer sizes/counts, stack sizes, etc.
-
-
-
-Qualification Testing
-*********************
-
-PICS Features
-=============
-
-The PICS features for each supported protocol & profile can be found in
-the following documents:
+Zephyr includes a complete Bluetooth Low Energy stack from application to radio
```

Comment:
Nit: I wouldn't say "application" (even though there is a few sample apps.

Reply:
I would actually prefer to leave as-is. We do provide some apps that, combined, can make a real application.